### PR TITLE
Add license to artifacts and src

### PR DESCRIPTION
**Pull request by ShiinaSekiu**

### CODE REVIEW

**Comment by CoelacanthusHex** [comment]
There are unrelated commits, please rebase onto main.

File: scripts/build.sh [hunk 1]
```diff
@@ -50,6 +50,10 @@ case "${1:-}" in
 make)
     cmake "$WORKSPACE"
     make -j
+    cp "$WORKSPACE/src/LICENSE" output/lpac-LICENSE
+    cp "$WORKSPACE/euicc/LICENSE" output/libeuicc-LICENSE
```
Comment (CoelacanthusHex) [inline scripts/build.sh:54]: Do you forget to upload LICENSE files for /src and /euicc?

**Comment by septs** [comment]
No need to copy for each switch case

```bash
mkdir -p "$BUILD/output"
cp "$WORKSPACE/src/LICENSE" "$BUILD/output/lpac-LICENSE"
cp "$WORKSPACE/euicc/LICENSE" "$BUILD/output/libeuicc-LICENSE"
cp "$WORKSPACE/cjson/LICENSE" "$BUILD/output/cjson-LICENSE"
cp "$WORKSPACE/dlfcn-win32/LICENSE" "$BUILD/output/dlfcn-win32-LICENSE"
```

File: scripts/build.sh (hunk 1 above)
Comment (ShiinaSekiu) [reply scripts/build.sh:54]: It's already in the project, it's always been there.

**Pull request merged**